src/main/scala/mesosphere/marathon/api/v2/AppsResource.scala

```diff
@@ -215,24 +215,11 @@ class AppsResource @Inject() (
     val appId = id.toRootPath
     val newVersion = Timestamp.now()
     def setVersionOrThrow(opt: Option[AppDefinition]) = opt
-      .map(_.copy(version = newVersion))
+      .map(app => app.copy(versionInfo = AppDefinition.VersionInfo.NoVersion))
```

Comment:
`setVersionOrThrow` now doesn't set the version anymore. At this point, it's not clear to me whether `NoVersion` (with a `version = Timestamp(0)`) actually is wanted, or overridden later on.